aiidalab/app.py

```diff
@@ -667,16 +677,16 @@ def check_path_exists_changed():
 
         self._started = True
 
-    def stop(self):
+    def stop(self) -> None:
         """Stop watching the app repository for file system events."""
         if self._monitor_thread is not None:
             self._monitor_thread_stop.set()
 
-    def is_alive(self):
+    def is_alive(self) -> Any:
```

Comment:
I tried using `bool` here but mypy complained. I don't understand why, I would think that the `and` operator always returns bool?

Reply:
Ooh, interesting. `bool | None` seems to work in this case.